src/search_options.hpp

```diff
@@ -92,19 +92,9 @@ struct LambdaOptions : public SharedOptions
     bool            computeLCA  = false;
     seqan3::genetic_code geneticCodeQry;
 
-    enum class ExtensionMode : uint8_t
-    {
-        AUTO,
-        XDROP,
-        FULL_SERIAL,
-        FULL_SIMD
-    };
-    // ExtensionMode   extensionMode = ExtensionMode::AUTO;
-#ifdef SEQAN_SIMD_ENABLED
-    ExtensionMode   extensionMode = ExtensionMode::FULL_SIMD;
-#else
-    ExtensionMode   extensionMode = ExtensionMode::FULL_SERIAL;
-#endif
+    #ifndef SEQAN_SIMD_ENABLED
+    #error "Lambda must be built with at least SSE4 support. Add -march=native to your compiler flags."
+    #endif
```

Comment:
This is not an ideal place for the error. I would place it at the top of search.cpp right after the header-includes.